gtk4-macros/src/blueprint.rs

```diff
@@ -0,0 +1,23 @@
+use anyhow::Result;
+use std::io::{Read, Write};
+use std::process::{Command, Stdio};
+
+pub(crate) fn compile_blueprint(blueprint: &[u8]) -> Result<String> {
+    let mut compiler = Command::new("blueprint-compiler")
+        .args(&["compile", "-"])
+        .stdin(Stdio::piped())
+        .stdout(Stdio::piped())
+        .spawn()?;
+
+    let mut stdin = compiler.stdin.take().unwrap();
+
+    stdin.write(b"using Gtk 4.0;\n")?;
```

Comment:
I think it isn't needed to explicitly write gtk version because 4.0 is the only possible version, but user will be able to write "using Adw 1;" manually (blueprint doesn't warn about duplicate `using` lines)

Reply:
Not sure this is needed at all... It is only useful for blueprint language server files to load the correct gir files and provide auto completion for you which is not something that is going to work in this case. Please drop it.

Reply:
> Not sure this is needed at all... It is only useful for blueprint language server files to load the correct gir files and provide auto completion for you which is not something that is going to work in this case. Please drop it.

For some reason it fails when I don't use 'using' line
![image](https://user-images.githubusercontent.com/63008755/211313606-7aff571d-353f-4a5d-8be8-6507278bf669.png)

Reply:
Sounds like a bug on blueprint side.

Reply:
Documentation says gtk declaration is required

![image](https://user-images.githubusercontent.com/63008755/211328865-54805b1a-a0a6-4d32-979e-9e4c0bec1b58.png)